### PR TITLE
Enable history output without requiring push

### PR DESCRIPTION
**Pull request by daved**
https://www.pivotaltracker.com/story/show/179074445

### CODE REVIEW

**Review by Naatan — CHANGES_REQUESTED** [review]
This seems a much larger logical change than necessary to address the bug. Remember we're at the polish stage; our changes should be as small and non-disruptive as possible.

**Comment by daved** [comment]
@Naatan I've swapped the meaning of "nil latest remote" to mean "all local", then explicitly set the "latest remote id" to the initial commit id so that all show as remote. This looks like a bigger change than it is because of indenting.

File: internal/runners/history/history.go [hunk 1]
```diff
@@ -54,22 +54,27 @@ func (h *History) Run(params *HistoryParams) error {
...
-			return locale.WrapError(err, "err_history_remote_branch", "Could not get branch by local branch name")
-		}
-
-		remoteCommitID := remoteBranch.CommitID
 		localCommitID := h.project.CommitUUID()
 
-		lastRemoteID, err = model.CommonParent(remoteCommitID, &localCommitID)
-		if err != nil {
-			return locale.WrapError(err, "err_history_common_parent", "Could not determine common parent commit")
+		if !h.project.IsHeadless() && h.project.Name() != "" {
```
Comment (Naatan) [inline internal/runners/history/history.go:68]: Why have `h.project.Name() != ""`? The `IsHeadless()` should already cover that.

**Review by Naatan — CHANGES_REQUESTED** [review]

File: internal/runners/history/history.go (hunk 1 above)
Comment (daved) [reply internal/runners/history/history.go]: From when I misunderstood how specifically "headless" was defined... fixed

**Review by Naatan — APPROVED** [review]

**Pull request merged**